build.gradle

```diff
@@ -91,39 +81,21 @@ subprojects { Project subproject ->
             apply plugin:"org.grails.grails-gsp"
         }
 
-        boolean usesGeb = project.name.contains('grails3-hibernate5-mongodb') || project.name.contains('grails3-mongodb') || project.name.contains('grails3-mongodb-database-per-tenant')|| project.name.contains('grails3-rxmongodb')
-
-        if (usesGeb) {
-            apply plugin:"com.github.erdi.webdriver-binaries"
-        }
-
         if (project.name == "examples-test-data-service") {
             apply plugin:"org.grails.grails-web"
         }
 
-        configurations.configureEach {
-            resolutionStrategy.eachDependency { DependencyResolveDetails details ->
-                if(details.requested.group == 'org.apache.groovy' && details.requested.name.startsWith('groovy')) {
-                    details.useVersion(groovyVersion)
-                } else if (details.requested.group == 'org.mongodb' && details.requested.module.name != 'mongodb-driver-rx') {
-                    details.useVersion(mongodbDriverVersion)
-                } else if (details.requested.group == 'org.springframework') {
-                    details.useVersion(springVersion)
-                } else if (details.requested.group == "org.springframework.boot") {
-                    details.useVersion(springBootVersion)
-                }
-            }
-        }
-
         dependencies {
-            testImplementation "jakarta.annotation:jakarta.annotation-api:$jakartaAnnotationApiVersion"
+            implementation platform("org.grails:grails-bom:$grailsVersion")
+            implementation "jakarta.servlet:jakarta.servlet-api"
```

Comment:
compileOnly?

Reply:
Good catch, I was able to remove it entirely, now that the test applications have updated dependencies.

Reply:
@jamesfredley I did not go through all the dependencies, to much of a maze for a quick review. I don't like when dependencies are assigned to subprojects with conditions. It becomes really hard to untangle. But we save that for another day 🌞